test/unit_tests/braket/default_simulator/test_density_matrix_simulator.py

```diff
@@ -860,3 +860,69 @@ def test_discontiguous_qubits_openqasm(discontiguous_qasm):
     measurements = np.array(result.measurements, dtype=int)
     assert len(measurements[0]) == 5
     assert result.measuredQubits == [0, 1, 2, 3, 4]
+
+
+def test_run_multiple():
+    payloads = [
+        OpenQASMProgram(
+            source=f"""
+            OPENQASM 3.0;
+            bit[2] b;
+            qubit[2] q;
+            {gates[0]} q[0];
+            {gates[1]} q[1];
+            b = measure q;
+            """
+        )
+        for gates in [("x", "z"), ("z", "x"), ("x", "x")]
+    ]
+    args = [[2], [5], [10]]
+    kwargs = [{"shots": 3}, {"shots": 6}, {"shots": 9}]
+    expected_measurements = [[1, 0], [0, 1], [1, 1]]
+    simulator = DensityMatrixSimulator()
+    for result, payload_args, expected in zip(
+        simulator.run_multiple(payloads, args=args), args, expected_measurements
```

Comment:
aren't you missing a `kwargs=kwargs` here?

Reply:
Ah I see what's going on. IMO this is pretty confusing, to allow `shots` to be in either